packages/falso/src/lib/rand-json.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  randBoolean,
+  randMovie,
+  randMovieCharacter,
+  randSinger,
+  randSnake,
+  randNumber,
+  randWord,
+} from '..';
+import { FakeOptions } from './core/core';
+
+export interface RandomJSONOptions extends FakeOptions {
+  min?: number;
+  max?: number;
+}
+
+const generateRandomValue = (): any => {
+  const availableValuesGenerators = [
+    randBoolean(),
+    randNumber(),
+    randNumber({ length: randNumber({ min: 1, max: 10 }) }),
+    randSinger(),
+    randSinger({ length: randNumber({ min: 1, max: 10 }) }),
+    randMovie(),
+    randMovie({ length: randNumber({ min: 1, max: 10 }) }),
+    randMovieCharacter(),
+    randMovieCharacter({ length: randNumber({ min: 1, max: 10 }) }),
+    randWord(),
+    randWord({ length: randNumber({ min: 1, max: 10 }) }),
+    randSnake(),
+    randSnake({ length: randNumber({ min: 1, max: 10 }) }),
+  ];
+
+  return availableValuesGenerators[
+    randNumber({ min: 0, max: availableValuesGenerators.length - 1 })
+  ];
+};
+
+const generateRandomObject = (length: number) => {
+  return randNumber({ length }).reduce(
+    (o, key) => ({ ...o, [key]: generateRandomValue() }),
+    {}
+  );
+};
+
+/**
+ * Generate a random JSON Object.
+ *
+ * @category Datatype
```

Comment:
@NetanelBasal I couldn't find a category that this would fit, so I've just created this new one, where we can then add new datatypes (boolean, string, float, int, etc) to it, what do you think?

Reply:
It's worth migrating them from `general` to here.